lms/djangoapps/instructor/management/commands/dump_userinfo.py

```diff
@@ -202,17 +234,33 @@ def handle(self, *args, **options):
             outfile = tempfile.NamedTemporaryFile(suffix='.csv', delete=False)
             outfile_name = outfile.name
 
-        csv_fieldnames = [label for field, label in CME_SPECIFIC_ORDER if len(label) > 0]
+        certificates, profiles, registrations, unpaid_registrations = self.query_database_for(course_id)
+        if 'openedx.stanford.djangoapps.register_cme' in settings.INSTALLED_APPS:
+            certificates, profiles, registrations, unpaid_registrations = self.query_database_for_new_app(course_id)
+
+        # If new_report option is passed generates, report with scores
+        if new_report:
+            # Add one new column to CSV for each calificable unit of each subsection
+            for profile in profiles:
```

Comment:
So, you are doing this for all the profiles, but the result is used only at the column names once. The easiest way is to use the first profile to get this extra fields, but I would like it better if there is a way to get the graded sections without users